infrastructure/kube/keep-prd/keep-dapp-token-dashboard-deployment.yaml

```diff
@@ -21,6 +21,6 @@ spec:
     spec:
       containers:
       - name: keep-dapp-token-dashboard
-        image: keepnetwork/token-dashboard:v1.12.0
+        image: keepnetwork/token-dashboard:v1.14.0
```

Comment:
Let's prepare a separate PR to increase the `master` `version` in `package.json` to `1.15.0-pre` and that changes this line to `v1.14.0` just like it is done here.